scripts/apps/search/components/fields/translations-list.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {IArticle} from 'superdesk-api';
+
+interface IProps {
+    svc: any;
```

Comment:
Can you remove `svc` from props? It's never clear what it contains and it's a hassle to drag it around the component tree. Use `getService` instead to get the services you need at any level of the component tree.